package/MDAnalysis/transformations/positionaveraging.py

```diff
@@ -0,0 +1,102 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+
+
+"""
+Trajectory Coordinate Averaging --- 
+=================================
+
+Averages the coordinates of a given trajectory with the N previous frames.
+For frames < N, the average of the frames iterated up to that point will be
+returned. 
+
+.. autoclass:: PositionAverager
+
+"""
+from __future__ import absolute_import
+
+import numpy as np
+import warnings
+
+
+
+class PositionAverager:  
+    """
+    
+    Averages the coordinates of a given timestep with the N previous frames. 
+    For frames < N, the average of the frames iterated up to that point will be
+    returned.  
+    
+    Example
+    -------
+    
```

Comment:
The example needs to be in a code block. Also add some text to explain what happens and what the parameters mean. 

How do you use it? What could you meaningfully compute in the example? People want to see what this can do for them and the best way to show it is by example (or examples).